pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -363,6 +363,14 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 					InputType:    InputTypeText,
 					PropertyName: "details",
 				},
+				{
+					Label:        "API URL",
+					Description:  "API URL of your Pager Duty account",
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Placeholder:  "{{ .ExternalURL }}",
```

Comment:
this should be taken from `pagerduty.DefaultAPIURL`.

you can change it to the default URL and then update
once you import the new version of alerting module.

Reply:
If we are going with URL, then I think this prop should be DefaultURL, will be using that.